surveys/sphinx-survey-2025.yml

```
@@ -171,16 +179,38 @@ questions:
 
 # Motivation: Understand the constellation of tools for documentation this responder works with.
 - title: "Which Python docstring style do you use?"
+  description: >-
+    Select N/A if you don’t use Python or Python docstrings.
+
+    Google-style docstrings look like:
+    ```rst
+    Arguments:
+        my_param (type): description
+    ```
+
+    Numpydoc-style docstrings look like:
+    ```rst
+    Arguments
+    ---------
+    my_param : type
+        description
+    ```
+
+    Sphinx-style docstrings look like:
+    ```rst
+    :param [my_param]: description
+    ```
   multiple: true
   other: true
   choices:
-  - "Google format (my_param (type): description)"
-  - "Numpydoc format (my_param : type \n description indented on new line)"
-  - "Sphinx format (:param [my_param]: description)"
+  - "Google format"
+  - "Numpydoc format"
+  - "Sphinx format"
   - "Markdown"
   - "I don’t know"
+  - "N/A"
```

```suggestion
  - "I do not write docstrings"
```
Feels we should be more explicit here